apps/docsite/sidebars.ts

```diff
@@ -51,6 +51,7 @@ const sidebars: SidebarsConfig = {
       ]
     },
     'Testing Unpublished Library Version',
+    'BundleSize'
```

Comment:
rename to Bundle Size

Reply:
This won't affect what displays on the website... This is just for mapping in docusaurus... On the website, it shows up as 'Bundle Size'

Reply:
what shows in the sidebar

Reply:
'Bundle Size'

Reply:
![image.png](https://github.com/microsoft/fluentui-charting-contrib/assets/35366351/0393ad79-7a2e-419a-83e2-7405685ecad3)